staging/src/k8s.io/client-go/rest/config.go

```diff
@@ -313,7 +318,7 @@ func DefaultKubernetesUserAgent() string {
 func InClusterConfig() (*Config, error) {
```

Comment:
Update the above comment to mention `ErrNotInCluster`

Reply:
fixed